enhancements/pause-and-resume-scans.md

```diff
@@ -0,0 +1,338 @@
+```
+
+Additionally, cluster administrators can remove the annotation to resume the scan:
+
+```console
+$ oc -n openshift-compliance annotate compliancescans/cis-ocp4-worker compliance.openshift.io/pause-
+```
+
+### Implementation Details/Notes/Constraints [optional]
+
+Under the hood, the Compliance Operator will handle the pause and resume logic
+in the `Reconcile` loop of the `ComplianceScan` object.
+
+If a `ComplianceScan` is paused, the Compliance Operator will determine the
+correct rerunner job, which is a `CronJob`, and suspend it (`suspend=true`).
+When the `ComplianceScan` is resumed, the Compliance Operator will ensure the
+`CronJob` rerunner is not suspended (`suspend=false`).
+
+An alternative approach would be to expose a formal attribute of the
+`ComplianceScan` Custom Resource Definition, like `pause` boolean attribute.
+However, since the existing rescan functionality uses annotation, we have an
+opportunity to keep pause and resume functionality consistent. Annotations are
+also documented as good ways for end users (cluster administrators in this
+case) to interact with non-standard features and behaviors.
+
+### Risks and Mitigations
+
+From a security perspective, it's not ideal to pause compliance scans. However,
+providing the ability to easily pause and resume scans through the operator
+does make the overall experience more palatable. In some cases, where clusters
+are going through significant changes or maintenance, pausing compliance scans
+is a more appropriate option than deleting the scan completely.
+
+From a user experience perspective, providing this functionality through an
+annotation is easier to consume than forcing users to look up `CronJob`
+rerunners for the scan their interested in and pausing it directly.
+
+## Design Details
+
+### Open Questions [optional]
+
+This is where to call out areas of the design that require closure before deciding
+to implement the design.  For instance,
+ > 1. This requires exposing previously private resources which contain sensitive
+  information.  Can we do this?
+
+
+
+### Test Plan
+
+This feature can be easily tested using our existing end-to-end test suite. We
+don't necessarily need to rely on testing the functionality of `CronJob`, but
+we can certainly test that the Compliance Operator updates the `suspend`
+attribute according to the annotation.
+
+We should have an end-to-end test that performs the following:
+
+1. Creates a new `ComplianceScan`, where the actual scan schedule doesn't necessarily matter
+2. Assert the `CronJob` associated with the `ComplianceScan` is set to `suspend=false`
+3. Pause the `ComplianceScan` using an annotation
+4. Assert the `CronJob` associated with the `ComplianceScan` is set to `suspend=true`
+5. Resume the `ComplianceScan` by removing the annotation
+6. Assert the `CronJob` associated with the `ComplianceScan` is set to `suspend=false`
+
+### Upgrade / Downgrade Strategy
```

Comment:
Is this section still applicable with annotations?

Reply:
I'm not sure if this fits into this section, but I guess any `pause=` annotation should be removed from `ComplianceScan` when downgrading to a version without pause/resume feature. And all the  `CronJob`s  should be switched `suspend=false`.

Reply:
That seems like a reasonable downgrade path.